src/twisted/internet/udp.py

```diff
@@ -365,6 +365,7 @@ def connectionLost(self, reason=None):
         """
         log.msg('(UDP Port %s Closed)' % self._realPortNumber)
         self._realPortNumber = None
+        self.maxThroughput = -1
```

Comment:
It's [even documented](https://github.com/twisted/twisted/blob/5c1777777f85aaf799c2269f9f5dddcf0d59803e/src/twisted/internet/udp.py#L66-L67).